<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowTextOverlayActionWidget</class>
 <widget class="QWidget" name="ShowTextOverlayActionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>147</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QCheckBox" name="fixedDurationCheckBox">
     <property name="text">
      <string>Display text for:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="customTextCheckBox">
     <property name="text">
      <string>Use custom text:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPlainTextEdit" name="customTextEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="durationWidget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSpinBox" name="durationSpinBox">
        <property name="suffix">
         <string/>
        </property>
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="durationSuffixLabel">
        <property name="text">
         <string>seconds</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="useSourceCheckBox">
     <property name="text">
      <string>Also show on source camera</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
